package.json

```diff
@@ -55,11 +55,8 @@
       "prettier --write"
     ]
   },
-  "packageManager": "pnpm@7.30.0",
+  "packageManager": "pnpm@8.2.0",
   "engines": {
-    "pnpm": "^7.13.0",
-    "yarn": "forbidden, use pnpm",
-    "npm": "forbidden, use pnpm",
```

Comment:
you can leave this in place if you'd like. you just need to switch `^` to `>=` which I believe is the thing that's currently stopping me from using this repo

Reply:
you can enable corepack locally to ease working with different package manager versions, or switch manually. until the issue with 8 is resolved, 7 must be used.

Reply:
added back engines.pnpm and rewriting it for node14 tests. left the other two removed as neither npm nor yarn check it anymore i think.